### PR TITLE
Add Italian translation

### PR DESCRIPTION
**Pull request by Giancky79**
traduzione in italiano

### CODE REVIEW

File: custom_components/renault/translations/sensor.it.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "stato": {
```
Comment (epenet) [inline custom_components/renault/translations/sensor.it.json:2]: This is a key, and should not be translated

File: custom_components/renault/translations/sensor.it.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "renault__plug_state": {
+      "unplugged": "Scollegato",
+      "plugged": "Collegato",
+      "plug_error": "Errore collegamento",
+      "plug_unknown": "Collegamento sconosciuto"
+    },
+    "renault__stato_ricarica": {
```
Comment (epenet) [inline custom_components/renault/translations/sensor.it.json:9]: This is a key, and should not be translated

File: custom_components/renault/translations/sensor.it.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      "not_in_charge": "Non in carica",
+      "waiting_for_a_planned_charge": "In attesa del programma ricarica",
+      "charge_ended": "Ricarica terminata",
+      "waiting_for_current_charge": "In attesa di corrente per ricarica",
+      "energy_flap_opened": "Sportello ricarica aperto",
+      "charge_in_progress": "Ricarica",
+      "charge_error": "Non in carica o Scollegato",
+      "unavailable": "Non disponibile"
+    },
+    "renault__modalità_ricarica": {
```
Comment (epenet) [inline custom_components/renault/translations/sensor.it.json:19]: This is a key, and should not be translated

**Review by epenet — COMMENTED** [review]
A few comments regarding the "keys" which must not be translated.
Will you also create "it.json"?

**Comment by epenet** [comment]
cc @VaguerOdin

**Review by Giancky79 — COMMENTED** [review]
sorry, i'm translate on a copy of custom and not correct after all, sorry

**Comment by VaguerOdin** [comment]
> cc @VaguerOdin

Hi, as I said yesterday the only comment I would make Is about the correct Eng-Ita translation. (Of course for the parts that can be translated)

For "renault__plug_state" should be:
```
"unplugged": "Scollegata",
"plugged": "Collegata",
"plug_error": "Errore collegamento",
"plug_unknown": "Collegamento sconosciuto"
```

For "renault__charge_state" would be:
```
"not_in_charge": "Non in carica",
"waiting_for_a_planned_charge": "In attesa della ricarica in programma",
"charge_ended": "Ricarica terminata",
"waiting_for_current_charge": "In attesa dell'attuale ricarica",
"energy_flap_opened": "Sportello di ricarica aperto",
"charge_in_progress": "In carica",
"charge_error": "Errore: Non in carica o scollegata",
"unavailable": "Non disponibile"
```

For "renault__charge_mode" would be:
```
"always": "Istantanea",
"always_charging": "Istantanea",
"schedule_mode": "Programmata"
```
Translation would be useless if we translate badly. Sorry for the delay.
Jess

**Comment by epenet** [comment]
@Giancky79 can you please update the PR to take into account the recommendations from @VaguerOdin? (or explain why you don't think the recommendations are invalid)

**Comment by Giancky79** [comment]
Schermata 2021-07-16 alle 16 23 18

yesterday i had fix it but i can't merge it, make new request?

**Comment by epenet** [comment]
You don't need to worry about the workflow. I will enable it after you have updated the files in your branch.
Or @VaguerOdin can convert the comments to "code suggestions" that can be merged.

**Comment by Giancky79** [comment]
ok, that's all, we hope in a future italian traslation 😄

**Review by VaguerOdin — APPROVED** [review]

File: custom_components/renault/translations/it.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+  "config: {
```
Comment (epenet) [inline custom_components/renault/translations/it.json:2]: There is a missing quote:
```suggestion
  "config": {
```

Comment (Giancky79) [reply custom_components/renault/translations/it.json]: sorry

**Review by VaguerOdin — APPROVED** [review]

**Pull request merged**